### ISSUE TITLE
ToDo List App Tutorial Broken V11.0.0

### ISSUE THREAD

**Issue by ARogala** [AvaloniaUI/Avalonia#12339]
**Describe the bug**
Following the tutorial the app does not build after the Data Binding step: https://docs.avaloniaui.net/docs/next/tutorials/todo-list-app/wiring-up-the-views 

**To Reproduce**
Follow the tutorial up to Data Binding section it fails to build after binding the ItemsSource 

error says "Unable to resolve suitable regular or attached property ItemSource on type Avalonia.Controls:Avalonia.Controls.ItemsControl"

**Expected behavior**
Build not to fail

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 10 pro with Visual Studio 2022. Used Avalonia .Net Core MVVM App for C# project, selected .NET 6.0 LTS with ReactiveUI MVVM toolkit, and Avalonia Version 11.0.0-preview5 
- Version Avalonia 11.0.0

**Additional context**
Hope this is enough information and sorry if this is the wrong place to post or if i did something wrong on my end. I did try this 3 times and it is not working for me. Tutorial fails at same point every time. I copied and pasted the code from the tutorial to ensure no typos.

Also I checked the code for this tutorial on Github and looks like it is still the code for the older version of Avalonia

https://docs.avaloniaui.net/docs/next/tutorials/todo-list-app/  click "You can find the code for the completed application [here](https://github.com/grokys/todo-tutorial)." links to old code

https://github.com/grokys/todo-tutorial  Github link has old code

**Comment by ARogala** [comment]
Ok think I found the error. In the docs on this page: https://docs.avaloniaui.net/docs/next/tutorials/todo-list-app/wiring-up-the-views 

the attribute on ItemsControl should be Items not ItemsSource

Please change

<ItemsControl ItemsSource="{Binding ListItems}">
      <ItemsControl.ItemTemplate>
        <DataTemplate>
          <CheckBox Margin="4"
                    IsChecked="{Binding IsChecked}"
                    Content="{Binding Description}"/>
        </DataTemplate>
      </ItemsControl.ItemTemplate>
</ItemsControl>

To

<ItemsControl Items="{Binding ListItems}">
	<ItemsControl.ItemTemplate>
		<DataTemplate>
			<CheckBox Margin="4"
			       IsChecked="{Binding IsChecked}"
			       Content="{Binding Description}"/>
		</DataTemplate>
	</ItemsControl.ItemTemplate>
</ItemsControl>

That solved the above error on my end. Please verify and change the Tutorial

Thanks.

**Comment by timunie** [comment]
That's wrong tbh. In 11.0, the property is called ItemsSource. So if Items works on your end, you are still using 0.10.x. i highly recommend to update to 11.0 when starting and learning Avalonia.

**Comment by ARogala** [comment]
Ok sorry new to Avalonia,

but my ToDoList.csproj file says 

PackageReference Include="Avalonia.Desktop" Version="11.0.0-preview5"

so doesn't that mean i am using the latest version?

**Comment by timunie** [comment]
No we are at 11.0.1 already. 11.0 is not in preview anymore

**Comment by ARogala** [comment]
ok thank you. How do we select projects with the latest version?

VS 2022 does not give the option for 11.0.1

![image](https://github.com/AvaloniaUI/Avalonia/assets/11381086/3d1c786c-6e49-4a17-8519-66319020568e)

I updated the Avalonia extension VS2022 only lets you select version 0.10.18 or 11.0.0-preview5 

there is no option for 11.0.1

**Comment by timunie** [comment]
Templates are separate from the extension

https://docs.avaloniaui.net/docs/next/get-started/install#install-avalonia-ui-templates

**Comment by timunie** [comment]
@ARogala thanks for your feedback. The samples are here to make your live easier and to onboard new people easily. So, you are really welcome :-) 

Enjoy Avalonia
